### PR TITLE
Added Sci Fi Helmet.

### PR DESCRIPTION
**Pull request by McNopper**

### CODE REVIEW

File: 2.0/SciFiHelmet/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Sci Fi Helmet
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+
+## License Information
+
+["Sci Fi Helmet" by Michael Pavlovic](http://quixel.se/usermanual/quixelsuite/doku.php?id=ddo_samples)
```
Comment (pjcozzi) [inline 2.0/SciFiHelmet/README.md]: At quick glance, I'm not sure what the license is.  Do you know or can you confirm with the artist that we can use it here?

Comment (McNopper) [inline 2.0/SciFiHelmet/README.md]: Will try to contact this guy.

Comment (McNopper) [inline 2.0/SciFiHelmet/README.md]: On the website. the license is the following "CC Attribution-Share Alike 3.0 Unported"
https://creativecommons.org/licenses/by-sa/3.0/

Comment (McNopper) [inline 2.0/SciFiHelmet/README.md]: I think this is the guy: @pavmike

Comment (McNopper) [inline 2.0/SciFiHelmet/README.md]: I twittered him.

Working website:
http://pavlovich.cgsociety.org/

Comment (McNopper) [inline 2.0/SciFiHelmet/README.md]: We have his okay - see Twitter.

Comment (pjcozzi) [inline 2.0/SciFiHelmet/README.md]: Thanks very much for checking, @McNopper!

**Pull request merged**